Sources/tart/VM.swift

```diff
@@ -326,9 +326,14 @@ class VM: NSObject, VZVirtualMachineDelegate, ObservableObject {
     }
 
     // Storage
-    var devices: [VZStorageDeviceConfiguration] = [
-      VZVirtioBlockDeviceConfiguration(attachment: try VZDiskImageStorageDeviceAttachment(url: diskURL, readOnly: false))
-    ]
+    let attachment: VZDiskImageStorageDeviceAttachment = try getStorageDevice(diskURL: diskURL, useFsWorkAround: vmConfig.os == .linux)
```

Comment:
Small nit-pick about passing `useFsWorkAround` since we [only support macOS 13+ already](https://github.com/cirruslabs/tart/blob/ac5f794e6d69652ee4841562c8c94e23df7ae6c2/Package.swift#L7). When looking at `getStorageDevice` alone it was not trivial for me to figure out when this workaround coming from. What do you think of inlining the method via ternary operator. IMO it improves readability. What do you think?

```suggestion
    let attachment: VZDiskImageStorageDeviceAttachment = vmConfig.os == .linux ?
      // Use cached caching mode for virtio drive to prevent fs corruption on linux when possible
      try VZDiskImageStorageDeviceAttachment(url: diskURL, readOnly: false, cachingMode: .cached, synchronizationMode: .full) :
      try VZDiskImageStorageDeviceAttachment(url: diskURL, readOnly: false)
```

Reply:
Added as you want. 🙂